x/campaign/types/total_supply_test.go

```diff
@@ -60,3 +60,47 @@ func TestUpdateTotalSupply(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateTotalSupply(t *testing.T) {
+	newCoins := func(coinsStr string) sdk.Coins {
```

Comment:
FYI, https://github.com/tendermint/spn/pull/520 will introduce a package `testutil/constructor` to centralized methods for this type of initialization

We can see depending on the first PR merged

Reply:
Can be used now :)

Reply:
rebased on `develop` and used `tc.Coins()` also here :)